src/txn/txn.c

```diff
@@ -2915,21 +2919,23 @@ __wt_verbose_dump_txn(WT_SESSION_IMPL *session)
      * handles is not thread safe, so some information may change while traversing if other threads
      * are active at the same time, which is OK since this is diagnostic code.
      */
-    WT_STAT_CONN_INCR(session, txn_walk_sessions);
     for (i = 0, s = txn_global->txn_shared_list; i < session_cnt; i++, s++) {
-        WT_STAT_CONN_INCR(session, txn_sessions_walked);
```

Comment:
As we're in __wt_verbose_dump_txn perf isn't really an issue so i'd just leave the individual increments that way we don't need to introduce an  err: label and WT_ERR macros.